sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -467,6 +467,14 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * <li>`pathGlobFilter`: an optional glob pattern to only include files with paths matching
    * the pattern. The syntax follows <code>org.apache.hadoop.fs.GlobFilter</code>.
    * It does not change the behavior of partition discovery.</li>
+   * <li>`modifiedBefore`: an optional timestamp to only include files with
+   * modification times  occurring before the specified time.  The provided timestamp
+   * must be in the following form:  <code>YYYY-MM-DDTHH:mm:ss</code>  Example:
+   * <code>2020-06-01T13:00:00</code>
+   * <li>`modifiedAfter`: an optional timestamp to only include files with
+   * modification times occurring after the specified time.  The provided timestamp
```

Comment:
ditto: `time.  The`

Reply:
This has been created in readwriter.py and also this file as well.